src/components/groups/group.middlewares.ts

```diff
@@ -105,7 +105,10 @@ export const validateGroup = () => {
       .isAfter()
       .withMessage('Múltbéli kezdéssel csoport nem hozható létre')
       .custom((value, { req }) => new Date(value).getTime() < new Date(req.body.endDate).getTime())
-      .withMessage('A kezdés nem lehet korábban, mint a befejezés'),
+      .withMessage('A kezdés nem lehet korábban, mint a befejezés')
+      .custom((value, { req }) => 
+        new Date(req.body.endDate).getTime() - new Date(value).getTime() <= 5*3600*1000)
```

Comment:
I think using the `date-fns`' `[differenceInHours](https://date-fns.org/v2.14.0/docs/differenceInHours)` is a bit more clean and safe solution.